homeassistant/components/rainmachine/config_flow.py

```diff
@@ -188,6 +189,12 @@ async def async_step_init(
                         CONF_USE_APP_RUN_TIMES,
                         default=self.config_entry.options.get(CONF_USE_APP_RUN_TIMES),
                     ): bool,
+                    vol.Optional(
+                        CONF_ALLOW_INACTIVE_ZONES_TO_RUN,
+                        default=self.config_entry.options.get(
+                            CONF_ALLOW_INACTIVE_ZONES_TO_RUN
+                        ),
+                    ): bool,
```

Comment:
This PR introduces a new feature, this cannot be part of a patch release.

Reply:
My bad; thank you!